Content.Shared/Xenoborgs/Components/CollideXenoborgFactoryComponent.cs

```diff
@@ -0,0 +1,16 @@
+namespace Content.Shared.Xenoborgs.Components;
+
+/// <summary>
+/// Valid items that collide with an entity with this component
+/// will begin to be reclaimed.
+/// <seealso cref="XenoborgFactoryComponent"/>
+/// </summary>
+[RegisterComponent]
+public sealed partial class CollideXenoborgFactoryComponent : Component
+{
+    /// <summary>
+    /// The fixture that starts reclaiming on collision.
+    /// </summary>
+    [DataField("fixtureId")]
```

Comment:
```suggestion
    [DataField]
```
redundant
see the infor message your IDE should show you